gr-filter/lib/fir_filter.cc

```diff
@@ -110,7 +110,7 @@ namespace gr {
       float
       fir_filter_fff::filter(const float input[])
       {
-	const float *ar = (float *)((unsigned long) input & ~(d_align-1));
+	const float *ar = (float *)((size_t) input & ~(d_align-1));
```

Comment:
That's not the correct type either.
You can build a 64 bit arch with a 32 bit size_t if you really want.

uintptr_t is the proper type here to get a type that's exactly as wide as void * and allows for integer math.